coalib/results/AbsolutePosition.py

```diff
@@ -27,6 +27,90 @@ def __init__(self,
     def position(self):
         return self._position
 
+    def __le__(self, other):
```

Comment:
for such scenarios there's a python built-in generator: https://docs.python.org/3.5/library/functools.html#functools.total_ordering